Include/internal/pycore_cpuinfo.h

```diff
@@ -0,0 +1,26 @@
+#ifndef Py_INTERNAL_CPUINFO_H
+#define Py_INTERNAL_CPUINFO_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef Py_BUILD_CORE
+#  error "this header requires Py_BUILD_CORE define"
+#endif
+
+#include <stdbool.h>
+
+typedef struct {
+    bool sse, sse2, sse3, sse41, sse42, avx, avx2, avx512vbmi;
```

Comment:
It's suggested that you define `sse` and `and` separately, which will be more intuitive.
```
bool sse, ...;
bool avx, ...;
```

Reply:
Yes and I'll add some comments.